.codeclimate.yml

```diff
@@ -1,4 +1,5 @@
 exclude_paths:
 - vendor/**/*
 - ZeroBin/**/*
-- tests/**/* 
```

Comment:
I researched on this thing...they ('*' & '**') are both same and does same work in this case for sure